src/utils/nftRewards.ts

```diff
@@ -202,3 +208,44 @@ export function tiersEqual({
     tier1.name === tier2.name
   )
 }
+
+// converts txNftArg[] into JB721TierParams[] (see juice-nft-rewards:structs/JB721TierParams.sol)
+export function buildJB721TierParams(nftRewards: TxNftArg) {
+  return Object.keys(nftRewards).map(cid => {
+    const contributionFloorWei = parseEther(
+      nftRewards[cid].contributionFloor.toString(),
+    )
+    const maxSupply = nftRewards[cid].maxSupply
+    const initialQuantity = maxSupply ?? DEFAULT_NFT_MAX_SUPPLY
+    const encodedIPFSUri = encodeIPFSUri(cid)
+
+    return {
+      contributionFloor: contributionFloorWei,
+      lockedUntil: BigNumber.from(0),
+      initialQuantity,
+      votingUnits: 0,
+      reservedRate: 0,
+      reservedTokenBeneficiary: constants.AddressZero,
+      encodedIPFSUri,
+      allowManualMint: false,
+      shouldUseBeneficiaryAsDefault: false,
+    }
+  })
+}
+
+// returns util arg send to nft Tx's
+export function buildNftTxArg({
```

Comment:
whats the "nft tx"? Is there a more descriptive name that might work better here?